trail_renderer.py

```diff
@@ -0,0 +1,44 @@
+from ursina import *
+
+class TrailRenderer(Entity):
+    def __init__(self, color1, color2, thickness=5, target=None, length=5, destroy_on_stop=False, **kwargs):
+        super().__init__()
+        self.target = target
+        self.model = Mesh(
+            vertices=[(target.world_position) for i in range(length)],
```

Comment:
Using target.world_position sounds wrong. Shouldn't it be (0,0,0)?
Ususally you would parent the TrailRenderer to the target

Reply:
Usually is the key word. I set it to that to skip a step when making a new instance of TrailRenderer.

Reply:
I realize *I* was the one who made that design decision, oops. However, I've  changed it to use parent, which makes it \easier to use and also added most of your added features like length and start and end colors.

Reply:
Sounds good, thanks for approving!